app.rb

```diff
@@ -0,0 +1,218 @@
+      end
+    end
+  end
+end
+
+def prompt_create_person
+  puts ''
+  puts '================================================='
+  puts '=   TYPE A NUMBER TO SELECT A STUDENT/TEACHER   ='
+  puts '================================================='
+  puts ''
+  puts '1) Create a student'
+  puts '2) Create a teacher'
+  print ':> '
+end
+
+def user_input_person(input)
+  print 'Name:> '
+  name = gets.chomp
+  print 'Age:> '
+  age = gets.chomp
+  print 'Has parent permission [Y/N]:> '
+  parent_permission = gets.chomp
+  permission = false
+  permission = true if parent_permission.upcase == 'Y'
+  if input == 1
+    print 'Classroom:> '
+    classroom = gets.chomp
+    [name, age, classroom, permission]
+  elsif input == 2
+    print 'Specialization:> '
+    specialization = gets.chomp
+    [name, age, specialization, permission]
+  end
+end
+
+def create_person
+  prompt_create_person
+  input = gets.chomp.to_i
+  puts ''
+  params = user_input_person(input)
+  if input == 1
+    Student.new(params[2], params[1], params[0], parent_permission: params[3])
+  elsif input == 2
+    Teacher.new(params[2], params[1], params[0], parent_permission: params[3])
+  end
+  list_all_people
+  puts ''
+  puts 'Person created successfully!'
+  puts ''
+end
+
+def create_book
+  puts ''
+  puts '================================================='
+  puts '=                    NEW BOOK                   ='
+  puts '================================================='
+  puts ''
+  print 'Title:> '
+  title = gets.chomp
+  print 'Author:> '
+  author = gets.chomp
+  Book.new(title, author)
+  puts ''
+  puts 'Book created successfully!'
+  puts ''
+end
+
+def prompt_create_rental
+  puts ''
+  print 'SELECT BOOK TO RENT:> '
+  book_idx = gets.chomp.to_i
+  person_instances = list_all_people
+
+  print 'SELECT RENTER[ID]:> '
+  renter_id = gets.chomp.to_i
+  print 'Date:> '
+  date = gets.chomp
+  [person_instances, book_idx, renter_id, date]
+end
+
+def create_rental
+  book_instances = list_all_books
+  params = prompt_create_rental
+  puts ''
+  selected_book = nil
+  selected_person = nil
+  book_instances.each_with_index do |book, index|
+    selected_book = book if (params[1] - 1) == index
+  end
+
+  params[0].each_with_index do |person, _index|
+    selected_person = person if params[2] == person.id
+  end
+  Rental.new(params[3], selected_book, selected_person)
+  puts ''
+  puts 'Rental created successfully!'
+  puts ''
+end
+
+def prompt_main
+  puts ''
+  puts '=================================================='
+  puts '=   PLEASE CHOOSE AN OPTION BY TYPING A NUMBER   ='
+  puts '=================================================='
+  puts ''
+  puts '  1 - List of all books'
+  puts '  2 - List of all people'
+  puts '  3 - Create a person'
+  puts '  4 - Create a book'
+  puts '  5 - Create a rental'
+  puts '  6 - List od all rentals for a given id'
+  puts '  7 - Exit'
+  print '  :> '
+end
```

Comment:
- [ ] Great job . Please like the previous reviewer noted and as clearly stated in this milestone requirement, It would be nice to move the input messages to a `main.rb` file, the main.rb is the entry point of your app and all list of options for the user to select from should be added there. Please, this will make your work more professional

![Screenshot_45](https://github.com/wacoo/school-library-system/assets/65298048/9d9103ef-e3bf-4879-b2fa-47c3f8b79655)

Reply:
I don't understand what you mean. I already did that.

Reply:
The code snippet above is from an earlier commit.

Reply:
Sorry, I forgot to push